### PR TITLE
react tic-tac-toe

### PR DESCRIPTION
**Pull request by suneethaykori**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 07week/tic-tac-toe/src/App.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+          gameOver: true
+        })
+      }
+    }
+
+  render() {
+    return (
+      <div id="board">
+        <div id="status">{this.state.gameOver? "Game Over!" : "Tic Tac Toe"}</div>
+        <div class="board-row">
```
Comment (stephsingler) [inline 07week/tic-tac-toe/src/App.js]: You can't use 'class' anymore, you need to use className

File: 07week/tic-tac-toe/src/App.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+          <button onClick={() => this.handleClick(0)}> {this.state.squares[0]} </button>
```
Comment (stephsingler) [inline 07week/tic-tac-toe/src/App.js]: This is a lot of repetition. Whenever you are repeating yourself this much look for a simpler way to do it.

**Review by stephsingler — APPROVED** [review]
good start! You should add a feature to alert the player whose turn it is..and also a reset to play again. Right now I would have to refresh the page to play again.

**Comment by suneethaykori** [comment]
Made changes based on the review comments.

**Pull request merged**